platform/darwin/src/MGLShapeSource.h

```diff
@@ -210,6 +210,31 @@ MGL_EXPORT
  */
 @property (nonatomic, copy, nullable) NSURL *URL;
 
+#pragma mark Querying the source's features
```

Comment:
I think `-featuresMatchingPredicate:` falls neatly under the existing “Accessing a Source’s Content” section heading.